cockpit-podman.spec.in

```diff
@@ -11,7 +11,16 @@ BuildRequires:  libappstream-glib
 
 Requires:       cockpit-bridge >= 138
 Requires:       cockpit-shell >= 138
+%if 0%{?rhel}
+Requires:       podman >= 1.2.0
+%else
 Requires:       podman >= 1:1.2.0
+%endif
+
+# we generally want CentOS packages to be like RHEL; special cases need to check %{centos} explicitly
+%if 0%{?centos}
+%define rhel %{centos}
+%endif
```

Comment:
Doesn't that only provide the macro from here on out? I can't find specifics in the documentation, but I would say that it needs to be put before the first use a couple of lines up.

Reply:
D'oh, my bad, thanks for spotting! Fixed.